wbce/framework/Insert.php

```diff
@@ -1258,6 +1258,13 @@ public function doFilter($sContent)
                 1
             );
         }
+        unset(
+            $this->_JsQueue,
+            $this->_CssQueue,
+            $this->_HtmlQueue,
+            $this->_TitleQueue,
+            $this->_MetaQueue
+        );
```

Comment:
This is much more elegant +1

Reply:
I actually liked your solution more, but we need to maintain the same functionality in both classes. Or at least, the Insert class shouldn't have less functionality than its static facade class I.

Regards, 
Christian